### PR TITLE
Add test for FakeCrashReporting

### PR DESCRIPTION
**Pull request by tomrozb**

### CODE REVIEW

File: lib-android/src/main/java/com/devinci/lib/crash/FakeCrashReporting.java [hunk 1]
```diff
@@ -15,7 +16,7 @@
   }
 
   @Override public void sendException(@NonNull Throwable throwable) {
-    // Does nothing intentionally
+    Log.e(FakeCrashReporting.class.getName(), "", throwable);
```
Comment (tomrozb) [inline lib-android/src/main/java/com/devinci/lib/crash/FakeCrashReporting.java:19]: Is there any way to test it with Robolectric?

Comment (arturdm) [inline lib-android/src/main/java/com/devinci/lib/crash/FakeCrashReporting.java:19]: Robolectric's own tests are a great source here:
https://github.com/robolectric/robolectric/blob/master/robolectric/src/test/java/org/robolectric/shadows/ShadowLogTest.java

Comment (tomrozb) [inline lib-android/src/main/java/com/devinci/lib/crash/FakeCrashReporting.java:19]: Great example. Added tests

File: lib-android-robolectric/src/main/java/com/devinci/lib/robolectric/Assertions.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package com.devinci.lib.robolectric;
+
+import org.robolectric.shadows.ShadowLog;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class Assertions {
+
+  public static void assertLogged(int type, String tag, String msg, Throwable throwable) {
```
Comment (arturdm) [inline lib-android-robolectric/src/main/java/com/devinci/lib/robolectric/Assertions.java:9]: We can extract this later to a separate assertj module. It's usage would look something like this:

``` java
LogItem logItem = ShadowLog.getLogs().get(0);
assertThat(logItem).hasType(type).hasTag(tag).hasMessage(msg).hasThrowable(throwable);
```

But it's ok for now.

**Comment by arturdm** [comment]
:+1:

File: lib-android-robolectric/src/main/java/com/devinci/lib/robolectric/Assertions.java (hunk 2 above)
Comment (tomrozb) [inline lib-android-robolectric/src/main/java/com/devinci/lib/robolectric/Assertions.java:9]: Created ticket on trello.

**Pull request merged**